crates/orchestrator/src/data_storage/aws_s3/mod.rs

```diff
@@ -60,7 +60,6 @@ impl DataStorage for AWSS3 {
         Ok(())
     }
 
-    #[cfg(test)]
```

Comment:
why do we need to remove this?

Reply:
So basically this function is needed in the e2e tests and tests environment was not available in e2e tests as it is a different crate ig so that's why made it public.